### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by daan-pay**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
     "postcss": "^8.0.3",
     "postcss-preset-env": "^6.7.0",
     "rollup": "^2.27.0",
-    "rollup-plugin-scss": "^2.6.0",
+    "rollup-plugin-scss": "^3.0.0",
     "rollup-plugin-stylelint": "^1.0.0"
```
Comment (michael-pay) [inline package.json:23]:
```suggestion
    "rollup-plugin-stylelint": "^1.0.0",
    "sass": "^1.43.5"
```
Adding sass as a local dep is favorable here, tested css regeneration with the file including that extra dependency and it works fine after upgrades.

**Review by michael-pay — CHANGES_REQUESTED** [review]
See my remark regarding the sass dependency

File: package.json (hunk 1 above)
Comment (daan-pay) [reply package.json]: I've added the sass dependency per your request.

**Review by michael-pay — APPROVED** [review]

**Pull request merged**